comp/api/api/apiimpl/grpc.go

```diff
@@ -10,13 +10,18 @@ import (
 	"fmt"
 	"time"
 
+	"github.com/DataDog/datadog-agent/comp/metadata/host/hostimpl/hosttags"
```

Comment:
@hush-hush I'm wondering looking at how we fetch host tags, if instead of having a dedicated package ir we would benefit from having `GetHostTags` function as part of the `/comp/metadata/host`. Maybe we can find ways to reduce the amount container related code we pull for fetching the host tags

Reply:
This comment is not a blocker on this PR is mostly to spark a conversation 😄